test/functionalTest/cases/117_convop_using_standard_ops/putIndividualContextEntity.test

```diff
@@ -0,0 +1,417 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp putIndividualContextEntity: PUT /v1/contextEntities/{entityId::id}
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Update/UPDATE E1/A1=1 and see it fail (PUT /v1/contextEntities/E1)
+# 02. Update/APPEND E1/A1=10 (POST /v1/updateContext)
+# 03. Update/UPDATE E1/A1=1 and see it work (PUT /v1/contextEntities/E1)
+# 04. Query CB for E1/A1
+# 05. Update/APPEND E2/A2=1 in CP1 
+# 06. Register E2/A2 in CB with CP1 as providing application
+# 07. Update/UPDATE E2/A2=22 in CB, making it being forwarded to CP1 (PUT /v1/contextEntities/E2)
+# 08. Query CP1 for E2/A2
```

Comment:
What about testing attributeFormat?

In addition, probably more tests

In addition, probably more tests could be added:

* !exist=entity::type
* exist=entity::type
* entity::type=TYPE